ql/ql/src/queries/style/FieldOnlyUsedInCharPred.ql

```diff
@@ -0,0 +1,25 @@
+/**
```

Comment:
As you're adding a new query here, does this need a change-note?

Reply:
QL-for-QL is different, it's not a product.  
No change-note or tests are needed.  
(The `Code scanning results / CodeQL` check is effectively a test).

Reply:
👍